src/styles/block/header.scss

```
@@ -0,0 +1,63 @@
+.menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
```

don't forget to create variables for all colors
```suggestion
  background: $c-white;
```